config/samples/numaplane.numaproj.io_v1alpha1_isbservicerollout.yaml

```diff
@@ -8,6 +8,6 @@ spec:
     spec:
       # Example from https://github.com/numaproj/numaflow/blob/main/examples/0-isbsvc-jetstream.yaml
       jetstream:
-        version: latest
+        version: 2.9.8
```

Comment:
Note to self: describe behavior seen with incompatibility of versions and consider upgrade strategy in those cases

Reply:
So, the behavior I was seeing was that the new StatefulSet would not come up on its own when going between 2.10 and 2.9. I confirmed from Derek that the backward incompatibility is expected to occur even when the pipelines are fully paused. He sent [this](https://docs.nats.io/release-notes/whats_new/whats_new_210#downgrade-warnings) link.

Reply:
in this case, we should probably always do delete and create/replace for isb upgrade? I'm going to create issue for it. and I think this needs to be addressed ASAP.

Reply:
https://github.com/numaproj/numaplane/issues/147

Reply:
I spoke again to Derek, and he said "users should just always use version 2.10+". So, I suppose maybe I shouldn't set this 2.9.8 after all in that case. It looks like if I use Numaflow 1.2.1, I have access to 2 different Jetstream versions in 2.10 that I could test with. Maybe I'll set it to 2.10.11?